source/screens/App/screens/LogIn/screens/LogIn.js

```diff
@@ -1,62 +1,66 @@
 import React, { Component } from 'react';
-import { USER_AUTH_TOKEN } from 'shared/const';
-import history from 'shared/utilities/history';
-import Auth from 'shared/api/Auth';
+import AuthService from 'shared/utilities/authService';
 import FormErrors from '../components/FormErrors';
 import './LogIn.css';
 
 export default class LogIn extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      email: '',
-      password: '',
-      formErrors: { email: '', password: '' },
-      emailValid: false,
-      passwordValid: false,
-      formValid: false,
-    };
-  }
+  state = {
+    email: '',
```

Comment:
Can you do this? define state outside of the constructor? I mean, I guess you can, but do we want to?